test/preparation/test_protonate.py

```diff
@@ -112,11 +125,15 @@ def test_protonate_peptide_with_pdb2pqr_metal():
 
     stru = sp.get_structure(test_pdb)
     prot.protonate_peptide_with_pdb2pqr(stru, 7.0, int_pdb, int_pqr)
-    assert list(map(lambda r:r.name ,stru.metalcenters[0].get_donor_mapper())) == ["GLU", "CYM", "CYM", "CYM"]
-    assert len(stru.atoms) == 5353 # removed 5 atoms
+    assert list(map(lambda r: r.name,
+                    stru.metalcenters[0].get_donor_mapper())) == [
+                        "GLU", "CYM", "CYM", "CYM"
+                    ]
```

Comment:
The same problem as above. This is making it hard to read

Reply:
Updated and fixed with new yapf settings